src/s3-copy.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env python3
```

Comment:
This is not needed and can cause problems if the Python executable is not in that path (i.e., if you used the UFS-RNR python stack of RDHPCS Hera and Orion).

Reply:
Henry, can you explain a bit more? Why is it not needed? I enjoy calling this as script as:
`s3-copy.py my.yaml`
If the python3 is not on the path you will be forced to call the script as:
`path-to-python/python3 s3-copy.py my.yaml`
which should override the header info.

Reply:
You are welcome to call this however you like. I explicitly launch my Python scripts as:

`python my_script.py`

Another point that you should consider is the `argparse` package. Here is an example:

This allows you to mitigate errors when passing arguments to a script. You can avoid errors possibly related to the order of the arguments being incorrect. This will be useful if/when you pass more arguments to your script.

Reply:
yeh. i was also looking at clic but decided that (since i am the only user) i can live with something simple for the first prototype. I am just starting with python. hopefully it will be cleaner in the future.